helm/ako/crds/ako.vmware.com_l4rules.yaml

```diff
@@ -0,0 +1,245 @@
+                        within the pool's list of available servers. Values LB_ALGORITHM_NEAREST_SERVER
+                        and LB_ALGORITHM_TOPOLOGY are only allowed for GSLB pool
+                      enum:
+                      - LB_ALGORITHM_LEAST_CONNECTIONS
+                      - LB_ALGORITHM_ROUND_ROBIN
+                      - LB_ALGORITHM_FASTEST_RESPONSE
+                      - LB_ALGORITHM_CONSISTENT_HASH
+                      - LB_ALGORITHM_LEAST_LOAD
+                      - LB_ALGORITHM_FEWEST_SERVERS
+                      - LB_ALGORITHM_RANDOM
+                      - LB_ALGORITHM_FEWEST_TASKS
+                      - LB_ALGORITHM_NEAREST_SERVER
+                      - LB_ALGORITHM_CORE_AFFINITY
+                      - LB_ALGORITHM_TOPOLOGY
+                      type: string
+                    lbAlgorithmHash:
+                      description: Criteria used as a key for determining the hash
+                        between the client and  server.
+                      enum:
+                      - LB_ALGORITHM_CONSISTENT_HASH_SOURCE_IP_ADDRESS
+                      - LB_ALGORITHM_CONSISTENT_HASH_SOURCE_IP_ADDRESS_AND_PORT
+                      - LB_ALGORITHM_CONSISTENT_HASH_URI
+                      - LB_ALGORITHM_CONSISTENT_HASH_CUSTOM_HEADER
+                      - LB_ALGORITHM_CONSISTENT_HASH_CUSTOM_STRING
+                      - LB_ALGORITHM_CONSISTENT_HASH_CALLID
+                      type: string
+                    minServersUp:
+                      description: Minimum number of servers in UP state for marking
+                        the pool UP.
+                      type: integer
+                    pkiProfileRef:
+                      description: Avi will validate the SSL certificate present by
+                        a server against the selected PKI Profile.
+                      type: string
+                    port:
+                      default: 80
+                      description: Port info
+                      maximum: 65535
+                      minimum: 1
+                      type: integer
+                    protocol:
+                      default: TCP
+                      description: Protocol info
+                      enum:
+                      - TCP
+                      - UDP
+                      - SCTP
+                      type: string
+                    sslKeyAndCertificateRef:
+                      description: Service Engines will present a client SSL certificate
+                        to the server.
+                      type: string
+                    sslProfileRef:
+                      description: When enabled, Avi re-encrypts traffic to the backend
+                        servers. The specific SSL profile defines which ciphers and
+                        SSL versions will be supported.
+                      type: string
+                    useServicePort:
```

Comment:
@swathinsankaran: On UI it maps to `DeactivatePortTranslation`. Customer might not able to understand `useServicePort` field uses due to its name? Also if value of it is true, it enables few other fields like `UseServiceSSLmode` and `ignoreServerPort`

Reply:
I have removed this field since no RFE is asking for this field.